core/server/master/src/main/java/alluxio/master/file/meta/InodeTreeBufferedIterator.java

```diff
@@ -302,4 +319,11 @@ public boolean hasNext() {
   public void remove() {
     throw new UnsupportedOperationException("remove is not supported in inode tree iterator");
   }
+
+  @Override
+  public void close() {
+    LOG.debug("Closing {} inode tree iterators", mActiveCrawlers.size());
+    mCoordinatorExecutor.shutdownNow();
```

Comment:
is this the correct order to shut them down?

Reply:
I don't think the order matters much. Both shutdown calls are asynchronous so the order of close does not make a big difference.